vllm/worker/model_runner.py

```diff
@@ -172,8 +172,11 @@ def load_model(self) -> None:
             )
 
         self.model_memory_usage = m.consumed_memory
+        logger_data = {
+            "model_memory_usage": f"{self.model_memory_usage / float(2**30):.4f}"
+        }
         logger.info("Loading model weights took %.4f GB",
-                    self.model_memory_usage / float(2**30))
+                    self.model_memory_usage / float(2**30), extra=logger_data)
```

Comment:
Continuing adding a variable from above...
```suggestion
                    model_memory_usage_gb, extra=logger_data)
```

Reply:
This is probably a worthwhile use of `logger_data`